tests/smoke/test_orchestrator_smoke.py

```diff
@@ -49,6 +49,29 @@ def test_orchestrator_different_epoch(
     assert isinstance(results.physical_activity_levels, models.Measurement)
 
 
+def test_orchestrator_none_epoch_ag_count(
```

Comment:
Given the ever-growing set of if-statements in the runner, this could do with a spy to assess that an expected function call is actually performed.